Libraries/Geolocation/Geolocation.ios.js

```diff
@@ -23,11 +23,9 @@ var subscriptions = [];
 var updatesEnabled = false;
 
 /**
- * /!\ ATTENTION /!\
- * You need to add NSLocationWhenInUseUsageDescription key
- * in Info.plist to enable geolocation, otherwise it's going
- * to *fail silently*!
- * \!/           \!/
+ * You need to include the `NSLocationWhenInUseUsageDescription` key
+ * in Info.plist to enable geolocation. Geolocation is enabled by default
+ * when you create a project with `react-native init`.
```

Comment:
@spicyj - good point, but do you know any cases when this or something similar happened? If so, I'd prefer simply having instructions on how to add the key rather than be a reason for unnecessary rejection.